_rules/heading-accessible-name-ffd0e9.md

```diff
@@ -0,0 +1,146 @@
+---
+id: ffd0e9
+name: Heading has accessible name
+rule_type: atomic
+description: |
+  This rule checks that each heading has an accessible name.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any HTML element with the [semantic role][] of `heading` that is [included in the accessibility tree][].
+
+## Expectation
+
+Each test target has a non-empty (`""`) [accessible name][].
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+Some assistive technologies may hide headings with empty [accessible name][] from the users. This depends both on the user agent and how the [accessible name][] was computed (the [accessible name and description computation][] is not clear concerning which characters should be trimmed) and of the assistive technology itself. Hence, there are cases where the outcome of this rule is _failed_, but users of certain assistive technology and browser combinations will not experience an issue.
+
+## Background
+
+- [Understanding Success Criterion 1.3.1: Info and Relationships][usc131]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `h1` element has a non-empty [accessible name][].
+
+```html
+<h1 aria-level="1">ACT rules</h1>
```

Comment:
I don't think the aria-level attribute is needed on an `<h1>` element. Like `role='heading'` it is redundant.

Reply:
Yes, my bad 😖